docs/schedule/index.md

```diff
@@ -5,7 +5,7 @@
 # GTFS Schedule Overview
 
 <div class="landing-page">
-    <a class="button" href="reference">Reference</a><a class="button" href="best-practices">Best Practices</a><a class="button" href="examples">Examples</a><a class="button" href="changes">Changes</a><a class="button" href="validate">Validate</a>
+    <a class="button" href="reference">Reference</a><a class="button" href="best-practices">Best Practices</a><a class="button" href="examples">Examples</a><a class="button" href="changes">Changes</a><a class="button" href="validate">Validate</a><a class="button" href="publish">Publish</a>
```

Comment:
Probably related to this discussion? 
https://mobilitydata-io.slack.com/archives/C06026D89G8/p1704812528356999

Adding a tab on the top lgtm.

Reply:
Yes.
After discussion, we're keeping this Publishing Resource as part of GTFS Schedule for now.